server/channels/app/platform/session.go

```diff
@@ -91,7 +93,9 @@ func (ps *PlatformService) ClearUserSessionCacheLocal(userID string) {
 }
 
 func (ps *PlatformService) ClearAllUsersSessionCacheLocal() {
-	ps.sessionCache.Purge()
+	if err := ps.sessionCache.Purge(); err != nil {
+		ps.Logger().Error("Failed to purge session cache", mlog.Err(err))
+	}
```

Comment:
Given that `ps.sessionCache.Purge` returns an `error` we should do the same with `ClearAllUsersSessionCacheLocal` and let the caller decide